tools/ubuntu-setup/misc.sh

```diff
@@ -10,3 +10,4 @@ sudo service ntp restart
 sudo ntpq -c lpeer
 
 sudo apt-get -y install git
+sudo apt-get -y install zip
```

Comment:
the original iOS artifact is `OpenWhiskIOSStarterApp.zip` coming from blue ant task. I did not want to make any semantic changes so I had to keep the file format. Without the ant zip task you need zip locally.